xain/grpc/test_coordinator_logic.py

```diff
@@ -75,14 +78,16 @@ def test_end_training():
 
 
 def test_end_training_round_update():
-    # Test that the round number is updated once all participants sent their updates
-    coordinator = Coordinator(required_participants=1)
+    # Test that the round number is updated once all participants send their updates
+    coordinator = Coordinator(required_participants=2)
     coordinator.on_message(coordinator_pb2.RendezvousRequest(), "peer1")
+    coordinator.on_message(coordinator_pb2.RendezvousRequest(), "peer2")
 
     # check that we are currently in round 1
     assert coordinator.round == 1
 
     coordinator.on_message(coordinator_pb2.EndTrainingRequest(), "peer1")
+    coordinator.on_message(coordinator_pb2.EndTrainingRequest(), "peer2")
```

Comment:
Between here add an assert that we're still in round 1

Reply:
Yes. The round is updated only after we get updates == required_participantes

Reply:
so assert it? See my suggestion.